server/app.js

```diff
@@ -45,6 +46,7 @@ app.use((req, res, next) => {
 app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
+app.use("/request", requestRouter);
```

Comment:
```suggestion
app.use("/requests", requestRouter);
```
Let's rename this route as per REST API naming convention